### ISSUE TITLE
Using with Kerberos available? 

### ISSUE THREAD

**Issue by WilixLead** [brianc/node-postgres#1443]
Hello! 

I use this module with pg-promise (as I found in their docs, pg-promise use this module by default)
I found in module docs "node-postgres uses the same environment variables as libpq to connect to a PostgreSQL server"
libpq support: 
PGKRBSRVNAME
PGREALM

for support kerberos authantication. I try use this envs, also I try use connection string like
"postgresql://myuser@localhost:5432/mydbname?krbsrvname=some/mydomain.com@MYREALM.COM"

But all not work. In most tries I got "password authentication failed for user..."
What I doing wrong?

Thanks!

**Comment by sehrope** [comment]
I think if you use the native driver then using Kerberos auth should work as it'd be deferring to libpq to handle everything. Make sure you're actually using the native driver though. By default you'd be using the pure JS driver.

See here for details: https://node-postgres.com/features/native

If you're using a third party library to wrap node-postgres then you'll have to look at the docs for the library to determine how to use enable the native driver.

**Comment by WilixLead** [comment]
I already try use native driver. Just this module and native all says "we support ENV and configuration from libpq".
libpq support this options by default. My problem is - I cant change my wrap lib and pg-promise it is just promises for this lib as I understand.

And thats why I try use standard libpq for kerberos authenticating.
Any way looks like node-postgres don't support this variables (they don't listed in lib/connection-parameters.js)

Thank you!

PS. I think, will be better fix documentation in connecting chapter
"node-postgres uses the same environment variables as libpq to connect to a PostgreSQL server"

**Comment by vitaly-t** [comment]
>  I cant change my wrap lib and pg-promise it is just promises for this lib as I understand.

No, it is  a lot more - automated connections, automated transactions, lots of helpers for generating complex insert and update queries, that's just to name a few.

pg-promise activates Native Binding via option `pgNative`: http://vitaly-t.github.io/pg-promise/module-pg-promise.html

**Comment by WilixLead** [comment]
Yes. I already try it. Got same result, but native module use native libpq support/communication. I leave them equal question. Waiting for help :)
